jest/setEnvVars.js

```diff
@@ -0,0 +1 @@
+process.env.NODE_ENV = 'test';
```

Comment:
is jest not getting the `NODE_ENV` from the `.env` file?  if not, we should probably use that instead to make it consistent.

Reply:
Yep. Good point.